### PR TITLE
Thumbnail api for msg draft

### PR DESCRIPTION
**Pull request by bitfriend**
Will fix [this issue](https://github.com/acterglobal/a3/issues/1250).

### CODE REVIEW

**Comment by bitfriend** [comment]
At this time upload of thumbnail is implemented.
Reading of thumbnail was already implemented.
`MsgContent::thumbnail_source()` and `MsgContent::thumbnail_info()`

**Review by gnunicorn — APPROVED** [review]
nice. just a request to make that API a bit more user friendly.

File: native/acter/api.rsh [hunk 1]
```diff
@@ -1190,13 +1193,19 @@ object MsgDraft {
     /// available for only image/video
     fn blurhash(value: string) -> MsgDraft;
 
+    /// available for only image/video/file/location
+    fn thumbnail_source(value: string) -> MsgDraft;
```
[the remark this answers is not in the document]
Comment (bitfriend) [reply native/acter/api.rsh:1197]: We have already implemented [getter fn](https://github.com/acterglobal/a3/blob/ccb0d52859552f2b7b61d286d14083add91e8d9f/native/acter/src/api/common.rs#L400) for thumbnail source.
At this time, I just implemented [setter fn](https://github.com/bitfriend/a3/blob/ccb0d52859552f2b7b61d286d14083add91e8d9f/native/acter/src/api/stream/msg_draft.rs#L400) for it.
I think both setter and getter should have the same name before we can avoid confusing at interpretation of meaning.

[the remark this answers is not in the document]
Comment (gnunicorn) [reply native/acter/api.rsh:1197]: I disagree. They need to be usable first and foremost. The `MsgDraft` is already different enough in many ways (like we give a path here but, which is read and uploaded _later_ and is _not_ the path we'll get back later), that there is no merit in them being "complementary" ... Quite the opposite, giving them the same name but then they are not two coins of the same thing might in itself be confusing: "why do I not get the same path out that I put in?" . I'd rather add more documentation and explain _what this does_ than expect the user of the API to know what other things might be related to it and have a similar name ...

[the remark this answers is not in the document]
Comment (bitfriend) [reply native/acter/api.rsh]: Okay, fixed by https://github.com/acterglobal/a3/pull/2199/commits/84ed032d8ea980627920bd22695d0d888f2b6b17

**Pull request merged**